server/service/src/sync/program_schemas/hiv_care_encounter.json

```diff
@@ -242,28 +242,28 @@
           "properties": {
             "bloodPressure": {
               "description": "75367002\tBlood pressure [mmHg]",
+              "examples": ["90/60", "120/80"],
```

Comment:
@mark-prins @richardmoizeau  do you think we should actually record the blood pressure in two values (systolic / diasystolic)? For example, we currently have a blood pressure graph that is broken. We should probably show both lines?

Reply:
@clemens-msupply, that what they do in the HPDB app - I don't think they require the blood pressure to be display in a graph.

Reply:
you mean, e.g. 090/060?

Reply:
recording the separate values would be good - I think it would be more helpful to have numerical values than a strangely formatted text value. We have options then for formatting as a string when needed, as well as the ability to plot on a chart